frontend/src/app/main/ui/workspace/sidebar/options/menus/measures.cljs

```diff
@@ -392,78 +467,140 @@
 
      (when (options :size)
        [:div {:class (stl/css :size)}
-        [:div {:class (stl/css-case :width true
-                                    :disabled disabled-width-sizing?)
-               :title (tr "workspace.options.width")}
-         [:span {:class (stl/css :icon-text)} "W"]
-         [:> numeric-input* {:min 0.01
-                             :no-validate true
-                             :placeholder (if (= :multiple (:width values)) (tr "settings.multiple") "--")
-                             :on-change on-width-change
-                             :disabled disabled-width-sizing?
-                             :class (stl/css :numeric-input)
-                             :value (:width values)}]]
-        [:div {:class (stl/css-case :height true
-                                    :disabled disabled-height-sizing?)
-               :title (tr "workspace.options.height")}
-         [:span {:class (stl/css :icon-text)} "H"]
-         [:> numeric-input* {:min 0.01
-                             :no-validate true
-                             :placeholder (if (= :multiple (:height values)) (tr "settings.multiple") "--")
-                             :on-change on-height-change
-                             :disabled disabled-height-sizing?
-                             :class (stl/css :numeric-input)
-                             :value (:height values)}]]
+        (if token-numeric-inputs
+          [:*
+           [:> numeric-input-wrapper*
+            {:disabled disabled-width-sizing?
+             :on-change on-width-change
+             :on-detach on-detach-token
+             :icon i/character-w
+             :min 0.01
+             :name :width
+             :property (tr "workspace.options.width")
+             :applied-tokens applied-tokens
+             :values values}]
+           [:> numeric-input-wrapper*
+            {:disabled disabled-height-sizing?
+             :on-change on-height-change
+             :on-detach on-detach-token
+             :min 0.01
+             :icon i/character-h
+             :name :height
+             :align :right
+             :property (tr "workspace.options.height")
+             :applied-tokens applied-tokens
+             :values values}]]
+          [:*
+           [:div {:class (stl/css-case :width true
+                                       :disabled disabled-width-sizing?)
+                  :title (tr "workspace.options.width")}
+            [:span {:class (stl/css :icon-text)} "W"]
+            [:> deprecated-input/numeric-input* {:min 0.01
```

Comment:
I suggest adding here a line break for follow the same indentation style for props as numeric-input-wrapper*